src/AddToCampaignHandler.php

```diff
@@ -245,7 +298,7 @@ public function addToCampaign($object, $campaignID)
         $request = $this->controller->getRequest();
         $message = _t(
             'AddToCampaign.Success',
-            'Successfully added {ObjectTitle} to {CampaignTitle}',
+            'Successfully added <strong>{ObjectTitle}</strong> to <strong>{CampaignTitle}</strong>',
```

Comment:
If you want to send back html:
 - set content-type to "text/html" below
 - Ensure you use convert::raw2xml() on all titles